state/pruner.go

```diff
@@ -122,7 +133,17 @@ func (p *Pruner) SetApplicationRetainHeight(height int64) error {
 	if currentAppRetainHeight > height || (!noCompanionRetainHeight && currentCompanionRetainHeight > height) {
 		return ErrPrunerCannotLowerRetainHeight
 	}
-	return p.stateStore.SaveApplicationRetainHeight(height)
+	if err := p.stateStore.SaveApplicationRetainHeight(height); err != nil {
+		return err
+	}
+	p.metrics.ApplicationBlockRetainHeight.Set(float64(height))
+	return nil
+}
+func (p *Pruner) checkHeightBound(height int64) bool {
```

Comment:
Please format the file. Ideally using gofumpt from your IDE to auto format on save.

Reply:
Weird, I had explicitly verified the other day that that is set up :/ Will check again.